multimodal/src/autogluon/multimodal/utils/environment.py

```diff
@@ -100,10 +100,10 @@ def infer_precision(
             warnings.warn(
                 "bf16 is not supported by the GPU device / cuda version. "
                 "Consider using GPU devices with versions after Amphere or upgrading cuda to be >=11.0. "
-                "MultiModalPredictor is switching precision from bf16 to 32.",
+                "MultiModalPredictor is switching precision from bf16 AMP to fp16 AMP.",
                 UserWarning,
             )
-            precision = 32
```

Comment:
I remember we set it to 32 on purpose because a model pretrained with bf16 may be difficult to converge if finetuned on fp16. bf16 can represent larger numbers than fp16, which can cause fp16 to overflow.

Reply:
thanks for the quick reply and explanation, will close with that 👍